examples/counter/package.json

```diff
@@ -22,24 +23,25 @@
   },
   "homepage": "https://github.com/wbuchwalter/ng2-redux#readme",
   "dependencies": {
-    "angular2": "^2.0.0-beta.0",
-    "ng2-redux": "^2.0.0",
-    "react": "^0.13.3",
-    "redux": "^3.0.0",
+    "es6-shim": "^0.35.0",
```

Comment:
Probably something to be dealt with later - but you might find this interesting re: es2-shim: https://github.com/rangle/rangle-starter/issues/86